.github/dependabot.yml

```
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: monthly
+  open-pull-requests-limit: 1
```

[suggestion] I think `1` is too few. How about increasing it to `5`?

```suggestion
  open-pull-requests-limit: 5
```